<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="3" parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test parallel="tests" name="Test2(failed)">
    <classes>
      <class name="testrunner.LoginTestRunner2">
        <methods>
          <include name="loginWithInvalidEmailTest"/>
          <include name="tearDown"/>
          <include name="Setup"/>
        </methods>
      </class> <!-- testrunner.LoginTestRunner2 -->
    </classes>
  </test> <!-- Test2(failed) -->
  <test parallel="tests" name="Test1(failed)">
    <classes>
      <class name="testrunner.LoginTestRunner1">
        <methods>
          <include name="loginTest"/>
          <include name="tearDown"/>
          <include name="Setup"/>
        </methods>
      </class> <!-- testrunner.LoginTestRunner1 -->
    </classes>
  </test> <!-- Test1(failed) -->
</suite> <!-- Failed suite [Suite] -->
